src/pages/index.tsx

```diff
@@ -73,6 +75,20 @@ function AuthShowcase() {
     { enabled: sessionData?.user !== undefined }
   );
 
+  const [email, setEmail] = useState("");
+  const handleResetPassword = async (e: FormEvent) => {
+    e.preventDefault();
+    console.log("reset password: ", email);
+    const { error } = await supabase.auth.resetPasswordForEmail(email, {
+      redirectTo: `${window.location.origin}/account/reset-password`,
+    });
+    if (error) {
+      alert(error.message);
+      return;
+    }
+    alert("Check your email for the password reset link");
```

Comment:
I think using a Toast / Notification from the UI library will be more elegant

Reply:
agreed!

however, this PR is mostly focused on implementing the authn password reset logic as opposed to composing the UI, so this wouldn't really be in scope here. hope that's alright!

Reply:
Yeap! No worries